src/main/java/htsjdk/samtools/util/AbstractProgressLogger.java

```diff
@@ -60,10 +61,18 @@ private synchronized void record() {
         if (this.lastChrom == null) readInfo = "*/*";
         else readInfo = this.lastChrom + ":" + fmt.format(this.lastPos);
 
+        final String rnInfo;
+        if(lastReadName != null) {
```

Comment:
Regarding the previous comments about formatting, I think we should be pushing the code to a consistent style based on the Sun/Oracle java style guidelines. Reformatting non-conforming code (such as the `if` on line 61) is preferable to committing new code that doesn't conform.

So here you'd add a space after `if` and delete the newline before `else`. IMO this is also an OK place to use a ternary op, but that's up to you.